src/web/modules/gateway_accounts/detail.njk

```diff
@@ -218,6 +214,33 @@
     </div>
   {% endif %}
 
+  {% if account.block_prepaid_cards === true %}
+    <div>
+      <h1 class="govuk-heading-s">Allow Prepaid Cards</h1>
```

Comment:
I've been told off before for not using sentence case https://design-system.service.gov.uk/styles/typography/#headings
```suggestion
      <h1 class="govuk-heading-s">Allow prepaid cards</h1>
```